support/ruby/collins-client/Gemfile.lock

```diff
@@ -1,50 +0,0 @@
-GEM
```

Comment:
We should keep the gemfile.lock in versioncontrol, and just run a `bundle update`. That way we know the exact versions of things that work when we are devving.